VideoWeb/VideoWeb/ClientApp/src/app/services/media-stream.service.ts

```diff
@@ -42,7 +42,7 @@ export class MediaStreamService {
         return from(this.navigator.mediaDevices.getUserMedia({ video: { deviceId: { exact: device.deviceId } } })).pipe(
             map(stream => {
                 const cloneStream = stream.clone();
-                if (this.videoFilterService.doesSupportVideoFiltering()) {
+                if (this.videoFilterService.doesSupportVideoFiltering() && this.videoFilterService.filterOn) {
```

Comment:
Could we get the unit tests for this service to cover the new cases?

Reply:
done